### ISSUE TITLE
conda and python compatibility error

### ISSUE THREAD

**Issue by Buetti85** [googlecolab/colabtools#3864]
Hello! I noticed that I can no longer make use of the AlphaFold application through Colab. The following error appears since last week. My assumption is that GoogleColab has updated their Python version which is now no longer compatible with Conda. Please see the error message below when trying to run an analysis via Colab. I would be really grateful if Google could solve the issue on their server as soon as possible. If the problem can be solved otherwise, I appreciate any input. Thank you!

WARNING:
    You currently have a PYTHONPATH environment variable set. This may cause
    unexpected behavior when running the Python interpreter in Miniconda3.
    For best results, please verify that your PYTHONPATH only points to
    directories of packages that are compatible with the Python interpreter
    in Miniconda3: /opt/conda
env: PATH=/opt/conda/bin:/opt/conda/bin:/opt/conda/bin:/opt/bin:/usr/local/nvidia/bin:/usr/local/cuda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/tools/node/bin:/tools/google-cloud-sdk/bin
Collecting package metadata (current_repodata.json): ...working... done
Solving environment: ...working... unsuccessful initial attempt using frozen solve. Retrying with flexible solve.
Collecting package metadata (repodata.json): ...working... done
Solving environment: ...working... unsuccessful initial attempt using frozen solve. Retrying with flexible solve.
Solving environment: ...working... 
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed

UnsatisfiableError: The following specifications were found
to be incompatible with the existing python installation in your environment:

Specifications:

  - conda==4.13.0 -> python[version='>=3.10,<3.11.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0']

Your python: python=3.11

If python is on the left-most side of the chain, that's the version you've asked for.
When python appears to the right, that indicates that the thing on the left is somehow
not available for the python version you are constrained to. Note that conda will not
change your python version to a different minor version unless you explicitly specify
that.

The following specifications were found to be incompatible with your system:

  - feature:/linux-64::__glibc==2.35=0
  - feature:|@/linux-64::__glibc==2.35=0

Your installed version is: 2.35

---------------------------------------------------------------------------
CalledProcessError                        Traceback (most recent call last)
[<ipython-input-3-a5fc66cf3ead>](https://localhost:8080/#) in <cell line: 22>()
     40       PATH=get_ipython().run_line_magic('env', 'PATH')
     41       get_ipython().run_line_magic('env', 'PATH=/opt/conda/bin:{PATH}')
---> 42       get_ipython().run_line_magic('shell', 'conda install -qy conda==4.13.0            && conda install -qy -c conda-forge              python=3.10              openmm=7.7.0              pdbfixer')
     43       pbar.update(80)
     44 

2 frames
[/usr/local/lib/python3.10/dist-packages/google/colab/_system_commands.py](https://localhost:8080/#) in check_returncode(self)
    135   def check_returncode(self):
    136     if self.returncode:
--> 137       raise subprocess.CalledProcessError(
    138           returncode=self.returncode, cmd=self.args, output=self.output
    139       )

CalledProcessError: Command 'conda install -qy conda==4.13.0            && conda install -qy -c conda-forge              python=3.10              openmm=7.7.0              pdbfixer' returned non-zero exit status 1.

**Comment by mayankmalik-colab** [comment]
Can you provide a min reproducible notebook that shows the error.?

**Comment by Buetti85** [comment]
Hi,

Thanks for your response. In the meanwhile the problem seems to have been solved in/by Colab. AlphaFold can be used again successfully for predictions without the incompatibility error. 

On an other note, I noticed that Colab GPU time is limited which is why I was planning to purchase additional GPU capacity (e.g., ColabPro). However, for some reason it is only possible to enter a post code from the US when finalizing the purchase. Is there a way to make the purchase from other countries, such as Switzerland?

Many thanks & best regards,
Pascal
